### ISSUE TITLE
Unable to change the attribute "id" on the parent div-element

### ISSUE THREAD

**Issue by dominikklein** [darrenjennings/vue-autosuggest#47]
First of all: thank you for a very useful component!

- `vue-autosuggest` version: 1.4.3
- `node` version: 10.5.0
- `npm` (or `yarn`) version: 6.2.0

Relevant code or config

```javascript
  data: () => ({
    component_attr_id_autosuggest: "autosuggest",
    component_attr_class_autosuggest__results_container: "autosuggest__results-container",
    component_attr_class_autosuggest__results: "autosuggest__results",
```

Problem description:
At the moment it is not possible to use a own 'id' for the parent div, because it is a fixed value in the data attributes. So if you add two vue-autosuggest fields on one page, you have a id duplication.

Suggested solution:
Add the possibility to set a own 'id' for the parent div and it whould be nice to have maybe a class on the parent div to have a better styling possibility.

**Comment by darrenjennings** [comment]
Thanks for the note. I agree this isn't ideal. I'll update it in a bit.

**Comment by djzara** [comment]
This is a really great update, but the PR was denied. Not to pester, but are there any updates on how and when that may be updated? Not being able to add multiple auto suggest components to a form requires a ton of workarounds, if you can get them to work in the first place. 

For example, say I have a set of results appear, no problem. I click the next field with an attached auto complete, no problem. However, now if I tab instead of click, and I enter data that returns results in that second field, both that field and the previous field open with the same exact data, which is not desirable.

If this is a separate issue, or even an implementation issue on my side, I do apologize. Thanks!

**Comment by darrenjennings** [comment]
@djzara I requested changes on the PR but they have not been addressed yet. I will try and get to it in the next week or so and just make the changes myself. Sorry it can't be sooner, just had a baby so haven't had as much time to devote! Stay tuned...

**Comment by djzara** [comment]
Sorry for the irrelevant(ish) comment, but I did want to say super grats on the baby :)  it's a wonderful experience!!

**Comment by darrenjennings** [comment]
fixed with v1.6.0 https://github.com/Educents/vue-autosuggest/releases/tag/v1.6.0

Documented under props:
https://github.com/educents/vue-autosuggest#props
